### DISCUSSION TITLE
Eigene reisen schneller finden

### DISCUSSION THREAD

**Discussion by EisenbahnfotografWesterwamd** [Traewelling/traewelling#2779] [Ideas / Feature Requests]
Moin! Nh schöne Sache währe es wenn man auf dem eigenen Account bzw. Auch bei anderen Accounts ein Datum aufschreiben könnte und dann von demjenigen oder von einem selbst die reisen sehen... Sprich das wenn eine Reise lange her ist man sie aber wieder finden will oder was ändern möchte das schneller funktioniert.

**Comment by pascalwittler** [comment]
Mögliche Überschneidung mit #2018

**Comment by EisenbahnfotografWesterwamd** [comment]
Hallo liebes träwelling Team,

ich wollte mal wissen ob es Jemals wieder ein normales Routing geben wird
oder ob es bei Luftline bleibt?

Kris ***@***.***> schrieb am Mo., 28. Apr. 2025, 21:42:
